configuration/pih/scripts/global/zl.js

```diff
@@ -1,74 +1,101 @@
+    var elem = document.getElementsByClassName("contacts");
+    var maxContacts = elem.length;
 
-  var hideOtherContacts = function () {
-    jq("#contact-2").hide();
-    jq("#contact-3").hide();
-    jq("#contact-4").hide();
-    jq("#contact-5").hide();
-    jq("#contact-6").hide();
-    jq("#contact-7").hide();
-    jq("#contact-8").hide();
-    jq("#contact-9").hide();
-    jq("#contact-10").hide();
-  };
+    var hideOtherContacts = function () {
+        jq("#contact-2").hide();
+        jq("#contact-3").hide();
+        jq("#contact-4").hide();
+        jq("#contact-5").hide();
+        jq("#contact-6").hide();
+        jq("#contact-7").hide();
+        jq("#contact-8").hide();
+        jq("#contact-9").hide();
+        jq("#contact-10").hide();
+    };
 
-  jq("#contact-" + contactsToShow).show();
-  hideOtherContacts();
-  jq("#show-less-contacts-button").hide();
+    jq("#contact-" + contactsToShow).show();
+    hideOtherContacts();
+    jq("#show-less-contacts-button").hide();
 
-  jq("#show-more-contacts-button").click(function () {
-    if (maxContacts > contactsToShow) {
-      contactsToShow++;
-      jq("#contact-" + contactsToShow).show();
-      if (contactsToShow > 1) {
-        jq("#show-less-contacts-button").show();
-      }
-      if (contactsToShow == 10) {
-        jq("#show-more-contacts-button").hide();
-      }
-    }
-  });
 
-  jq("#show-less-contacts-button").click(function () {
-    if (maxContacts > contactsToShow || contactsToShow == 10) {
-      if (contactsToShow > 1) {
-        jq("#contact-" + contactsToShow).hide();
-        contactsToShow--;
-        if (contactsToShow == 1) {
-          jq("#show-less-contacts-button").hide();
+    jq("#show-more-contacts-button").click(function () {
+        if (maxContacts > contactsToShow) {
+            contactsToShow++;
+            jq("#contact-" + contactsToShow).show();
+            if (contactsToShow > 1) {
+                jq("#show-less-contacts-button").show();
+            }
+            if (contactsToShow == 10) {
+                jq("#show-more-contacts-button").hide();
+            }
+        }
+    });
+
+    jq("#show-less-contacts-button").click(function () {
+        if (maxContacts > contactsToShow || contactsToShow == 10) {
+            if (contactsToShow > 1) {
+                jq("#contact-" + contactsToShow).hide();
+                contactsToShow--;
+                if (contactsToShow == 1) {
+                    jq("#show-less-contacts-button").hide();
+                }
+            }
+            jq("#show-more-contacts-button").show();
         }
-      }
-      jq("#show-more-contacts-button").show();
+    });
+
+
+    for (let i = 0; maxContacts > i; i++) {
+
+        // get Contact values
+        let contactValue = {}
+        jq(`#contact-${i} input, #contact-${i} select, #contact-${i} textarea, #contact-${i} radio`).each(function (i, obj) {
+            if (jq(obj).val().indexOf(5) && jq(obj).val().indexOf(448)) {
```

Comment:
Just out of curiosity, what is at index 5 and index 448?

Reply:
The default for Dead is 5 and also 448 for Serological status.